app/package.json

```diff
@@ -22,14 +23,20 @@
     "glob": "^9.3.2",
     "postcss": "^8.4.21",
     "postcss-import": "^15.1.0",
-    "prettier": "^2.8.4",
-    "prettier-plugin-tailwindcss": "^0.2.4",
-    "tailwindcss": "^3.2.7",
+    "prettier": "^3.0.2",
+    "prettier-plugin-jinja-template": "^1.0.0",
+    "prettier-plugin-tailwindcss": "^0.5.3",
+    "tailwindcss": "^3.3.3",
     "vite": "^4.1.4",
     "vite-plugin-pwa": "^0.14.4",
     "vite-plugin-static-copy": "^0.17"
   },
   "dependencies": {
-    "htmx.org": "^1.9.4"
+    "@wagmi/core": "^1.3.9",
+    "@web3modal/ethereum": "^2.7.1",
+    "@web3modal/html": "^2.7.1",
+    "htmx.org": "^1.9.4",
+    "viem": "^1.7.0",
+    "wallet-standard-list": "link:../../wallet-standard-list/wallet-standard-list"
```

Comment:
remind me to publish this onto npm, this probably makes prod builds unusable right now